code/game/objects/objs.dm

```diff
@@ -123,32 +123,34 @@ GLOBAL_LIST_EMPTY(objects_by_id_tag)
 		return null
 
 /obj/proc/updateUsrDialog()
-	if((obj_flags & IN_USE) && !(obj_flags & USES_TGUI))
```

Comment:
it is clearly used, you can't just remove it without understanding why it exists.

Reply:
"This flag is literally only used in two objects in the game and seemingly does nothing"
A bunch of objects that use TGUI, do not use this flag. Either this flag is obsolete, or there is something seriously wrong with most objects that use TGUI.

Reply:
I am pretty sure it's the former, in fact.